### ISSUE TITLE
List of filter fields

### ISSUE THREAD

**Issue by dvenprasad** [data-refinery/data-refinery#139]
To keep the prototype I'm building is consistent with the data available/extractable in the back end, here are the filter fields:
- Data Type/Technology (eg: RNA-seq, microarray)
- Gene Identifier
- Organism
- Platform
- Publication (Yes or no) 
- Submitter's Institution

@Miserlou @kurtwheeler @jaclyn-taroni : Would they be consistent? Are there some fields which will not be available for v1.1 but will be in the future?

**Comment by Miserlou** [comment]
I think we can do Organism and Platform as it stands. Technology is a trivial fix. Publication I don't know, sumitter's institution might be harder. 

Gene Identifier is an open issue since this requires adding an extra step to the Salmon pipeline - related: https://github.com/data-refinery/data-refinery/issues/131

**Comment by Miserlou** [comment]
Jackie says also related: https://github.com/data-refinery/data-refinery/issues/95

**Comment by Miserlou** [comment]
And, from my perspective, so is #140

**Comment by cgreene** [comment]
I'm relatively sure we will not be able to filter by submitter's institution. It may be very incomplete and/or not current with where people are now (which is probably what the use case targets).

**Comment by Miserlou** [comment]
Agreed, see my comment here: https://github.com/data-refinery/data-refinery/issues/140#issuecomment-367845805

**Comment by jaclyn-taroni** [comment]
I think we might be able to filter based on institution: [#140 (comment)](https://github.com/data-refinery/data-refinery/issues/140#issuecomment-368024327)

**Comment by jaclyn-taroni** [comment]
I'll note that, at first, we may want to restrict filtering by gene identifier to within an organism, e.g., if I select a single organism to filter by, then I can filter by gene.

**Comment by jaclyn-taroni** [comment]
Okay, here's a sample of ArrayExpress experiments that include RNA-seq and microarray data. I grabbed ten accession numbers from experiments that included "transcription profiling by array" AND "RNA-seq of coding RNA" in the Experiment type. Most of them are SuperSeries imported from GEO. I didn't come across any experiments that met this criterion that were not imported from GEO.

SuperSeries:

* [E-GEOD-66382](https://www.ebi.ac.uk/arrayexpress/experiments/E-GEOD-66382/)
* [E-GEOD-43970](https://www.ebi.ac.uk/arrayexpress/experiments/E-GEOD-43970/)
* [E-GEOD-71218](https://www.ebi.ac.uk/arrayexpress/experiments/E-GEOD-71218/)
* [E-GEOD-49893](https://www.ebi.ac.uk/arrayexpress/experiments/E-GEOD-49893/)
* [E-GEOD-61182](https://www.ebi.ac.uk/arrayexpress/experiments/E-GEOD-61182/)
* [E-GEOD-33588](https://www.ebi.ac.uk/arrayexpress/experiments/E-GEOD-33588/)
* [E-GEOD-32716](https://www.ebi.ac.uk/arrayexpress/experiments/E-GEOD-32716/)
* [E-GEOD-62389](https://www.ebi.ac.uk/arrayexpress/experiments/E-GEOD-62389/)
* [E-GEOD-54652](https://www.ebi.ac.uk/arrayexpress/experiments/E-GEOD-54652/)

Not a SuperSeries:

* [E-GEOD-16136](https://www.ebi.ac.uk/arrayexpress/experiments/E-GEOD-16136/) -- this one seems like an edge case to me?
